lib/swagger-client.js

```diff
@@ -39,10 +39,19 @@ class Root extends Component {
    */
   loadSpec() {
     return new Promise((resolve, reject) => {
-      this.http.request('GET', { path: '/swagger.json' }, (err, res) => {
+      this.http.request('GET', { headers: { accept: 'application/json' }, path: '/openapi/v2' }, (err, res) => {
         if (err) return reject(err);
-        if (res.statusCode !== 200) {
-          return reject(new Error(`Failed to get swagger.json: ${res.statusCode}`));
+        if (res.statusCode === 404) {
+          return this.http.request('GET', { path: '/swagger.json' }, (err, res) => {
```

Comment:
Can you run the linter?  I think it will complain about aliasing here (e.g., `err` in the first callback and `err` here).

Reply:
fixed `err` and `res` aliasing